source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +224,53 @@ export default class WindowManager {
       return
     }
 
+    const platformIcons: {[prop: string] : string} = {
+      'darwin': '/png/22x22_white.png',
+      'win32': '/icon.ico'
+    }
+
+    this._mainWindow.on('show', async () => {
+      if (this._tray == null) {
+        if (process.platform === 'linux') {
+          let size = '16'
+          const sizeList = ['16', '24', '32', '48', '64', '96', '128', '256', '512']
+          const display = screen.getPrimaryDisplay()
+          if (process.env.XDG_CURRENT_DESKTOP === 'GNOME') {
+            size = display.workArea.y + ''
+          } else if (process.env.XDG_CURRENT_DESKTOP === 'KDE'){
+            size = display.size.height - display.workArea.height + ''
+          }
+          if (sizeList.includes(size)) {
+            this._tray = new Tray(path.join(__dirname, `assets/icons/png/${size}x${size}.png`))
+          } else {
+            this._tray = new Tray(path.join(__dirname, `assets/icons/png/16x16.png`))
```

Comment:
16 is a very small fall-back. Most platforms have a task bar around 32. I suggest make 32.
Fails `yarn lint` : `error    Strings must use singlequote`
```typescript
            this._tray = new Tray(path.join(__dirname, 'assets/icons/png/32x32.png'))
```